src/routes.js

```diff
@@ -4,6 +4,8 @@ const bodyParser = require('body-parser');
 const jsonParser = bodyParser.json();
 const User = require('./User');
 
+// TODO: User's privileges should be fetched through a single API endpoint
+//       and not by matching client's routes
```

Comment:
Why is matching client routes a thing? You should be able to expose whatever endpoint and I just pass that URL to my requests. It is bad practice to match routes. In case of route changes in front-end, we will have to also change it in backend.